.drone-v1.yml

```diff
@@ -0,0 +1,252 @@
+---
+
+kind: pipeline
+type: kubernetes
+name: build
+
+steps:
+- name: Build Project
+  image: quay.io/ukhomeofficedigital/openjdk11:v11.0.5_10
+  commands:
+  - ./gradlew build
+  when:
+    event:
+      - push
+
+- name: Sonar Scanner
+  image: quay.io/ukhomeofficedigital/sonar-scanner:v3.0.3
+  when:
+    event:
+      - push
+  depends-on:
+    - Build Project
+
+- name: Build & Push
+  image: plugins/docker
+  settings:
+    registry: quay.io
+    repo: quay.io/ukhomeofficedigital/hocs-templates
+    tags:
+      - build-${DRONE_BUILD_NUMBER}
+      - ${DRONE_COMMIT_SHA}
+      - branch-${DRONE_COMMIT_BRANCH/\//_}
+  environment:
+    DOCKER_PASSWORD:
+      from_secret: QUAY_ROBOT_PASSWORD
+    DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+  when:
+    event:
+      - push
+  depends-on:
+    - Build Project
+
+- name: Build & Push Latest
+  image: plugins/docker
+  settings:
+    registry: quay.io
+    repo: quay.io/ukhomeofficedigital/hocs-templates
+    tags:
+      - master
+  environment:
+    DOCKER_PASSWORD:
+      from_secret: QUAY_ROBOT_PASSWORD
+    DOCKER_USERNAME: ukhomeofficedigital+hocs_quay_robot
+  when:
+    branch:
+      - master
+    event:
+      - push
+  depends-on:
+    - Build Project
+
+trigger:
+  event:
+    - push
+
+---
+
+kind: pipeline
+type: kubernetes
+name: deploy
+depends_on:
+  - build
+
+steps:
+- name: Clone Kube Repo
+  image: plugins/git
+  commands:
+    - git clone https://github.com/UKHomeOffice/kube-hocs-templates.git
+  when:
+    event:
+      - push
+      - promote
+
+- name: Deploy To CS-DEV
```

Comment:
It's canonically `cs-dev`, not `CS-DEV`.

Reply:
This has been rectified.